modules/references/references.go

```diff
@@ -33,7 +33,7 @@ var (
 	// issueNumericPattern matches string that references to a numeric issue, e.g. #1287
 	issueNumericPattern = regexp.MustCompile(`(?:\s|^|\(|\[|\')([#!][0-9]+)(?:\s|$|\)|\]|[:;,.?!]\s|[:;,.?!]$)`)
```

Comment:
Just a question, should we also add `"` support here?

Reply:
@charles7668 I think this is the last one we should fix in this PR.

Reply:
> @charles7668 I think this is the last one we should fix in this PR.

I fixed it